### PR TITLE
Add the filters icon to the search filters list on mobile

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-1944)
[this task](https://uktrade.atlassian.net/browse/ED-1945)

# mobile, no js

image

# mobile open, no filters applied

image

# mobile closed, no filters applied

image

# mobile opened, filters applied

image

# mobile closed, filters applied

image

# desktop, no filters applied

image

# desktop, filters applied

image

### CODE REVIEW

File: enrolment/static/images/icons/filters.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="30px" height="29px" viewBox="0 0 30 29" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
+    <title>Page 1</title>
+    <desc>Created with Sketch.</desc>
```
Comment (kowalcj0) [inline enrolment/static/images/icons/filters.svg]: @richtier Should we keep such meta-data?

Comment (richtier) [inline enrolment/static/images/icons/filters.svg]: I'll remove it

**Comment by kowalcj0** [comment]
@richtier 1 question about the pictures from the PR description, are they screenshots taken after you made the changes or they're are pix from the design?

**Comment by richtier** [comment]
@kowalcj0 they are screenshots taken after I made the change. The other would be cheating :)

**Pull request merged**